WPMedia/engine-theme-sdk
# TMEDIA-286 - Add placeholder space for PoWa Videos

…avoid cumulative layout shift

## Description
Update VideoPlayer's to show a placeholder space that the PoWa player will load into. This reduces CLS when PoWa is loading

## Jira Ticket
- [TMEDIA-286](https://arcpublishing.atlassian.net/browse/TMEDIA-286)

## Test Steps
1. Run Fusion news theme with SDK linked
2. Visit Video page and verify there is a black placeholder block before video loads - http://localhost/video/2020/11/17/thorns-on-the-stem-of-a-tree/?_website=the-gazette

## Effect Of Changes
### Before

Showing CLS with no placeholder space

![video-placeholder-before](https://user-images.githubusercontent.com/868127/124458497-f3c30d00-dd84-11eb-91ff-940904161377.gif)


### After

Reduced CLS

![video-placeholder](https://user-images.githubusercontent.com/868127/124458468-e7d74b00-dd84-11eb-9e95-5400f6d573ba.gif)

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
